apps/dashboard/src/lib/components/Org/AddOrgModal/AddOrgModal.svelte

```diff
@@ -95,9 +95,6 @@
 
       snackbar.success();
       await getOrganizations($profile.id);
-
-      resetForm();
-
```

Comment:
Please don't remove the resetForm, instead move it to go after `$newOrgModal.open = false`